src/device/mod.rs

```diff
@@ -246,6 +246,8 @@ pub struct Entities {
     // Nurse
     nurse: Option<Arc<Nurse>>,
 
+    aggregator: Option<Arc<ConnectivityDataAggregator>>,
```

Comment:
The comment above about no-op implementation also applies here

Reply:
The thing is that Derp may not always need to use Aggregator - e.g. I think there is no point in adding aggregator to derp in tcli imho

Reply:
Well, the derp inside Telio is normally created and I think we want to test aggregator with it, I thought about the separate derp instance, which isn't really connected to Nurse or anything, so I don't think we could use it for aggregation tests even if we wanted